applications/luci-app-babeld/luasrc/model/cbi/babeld.lua

```diff
@@ -0,0 +1,126 @@
+m = Map("babeld", "Babeld Overview", translate("Babeld Overview"))
```

Comment:
The translation is missing from the second argument in the function. This is the heading of the section. The question is then only do you need the third argument in the function. This is usually used to describe the section more precisely. Just have a look at the other map functions in lua in the repo.